### ISSUE TITLE
Add matching by name and address option matching for Bulk Upload

### ISSUE THREAD

**Issue by joemcl** [WorkingFamilies/votivate#97]
**Module**

Bulk Upload

**Detailed Description**

Add an option to bulk upload a list with First Name, Last Name and Address and match on those fields 

**Urgency**

1

Use case: I have a list of online ballot petition signers with first, last, and address (street, city, st, zip). I want to match those to voter names. As well, if I have cells or e-mails for some of the same names, add that to field mapping and match on those. Basically, add a capability similar to what Bulk Upload in VAN/Votebuilder is able to do. 

![6c2d76c8-c1a5-4919-ac4c-b19a83a644b9](https://user-images.githubusercontent.com/3912434/59823506-41c26080-92e3-11e9-9776-84e94c810c60.jpg)

**Comment by codygordon** [comment]
Big change to the bulk upload feature, but something we've been talking about implementing in the near future.

Name and address matching is pretty tricky, so this could take a bit.

**Comment by codygordon** [comment]
@rbrtmnz you've talked about this a few times – please chime in on what you want to see as part of the field mapping feature.

**Comment by joemcl** [comment]
yup @codygordon appreciate that it's definitely not easy, thanks so much. If I can I'll dive into VAN API and Votebuilder over the weekend and see what else can be gleaned. Also BTW Shai Sachs, who was an early OSDI organizer, and at NGPVAN for years and knows it's data stuff maybe better than anyone, left NGPVAN in February for a for-profit. Thinking worth seeing if he's interested in talking. See https://shaisachs.github.io/2019/02/24/leaving-ngp-van.html .

**Comment by rbrtmnz** [comment]
This seems nice but is not necessary and falls below other needs. I think if any change is going to be made to bulk uploading the priority needs to be mapping since it is currently a huge pain and time consuming to upload anything with multiple dispositions. Additionally, I would be hesitant to prioritize this against any of our needs when we have ample matching options. Joe if you send me your list and I can return it matched to the voter file. Quick turnaround.

**Comment by joemcl** [comment]
@rbrtmnz yeah cool, and the immediate use case I had is past, I ended up looking up one by one.
